src/api/manager/presets.ts

```diff
@@ -16,26 +16,30 @@ export async function getPresetByid(id: string): Promise<PresetWithId> {
 
 export async function getMultiviewPresets(): Promise<MultiviewPreset[]> {
```

Comment:
What's the difference between this function and the one in api/manager/multiview-presets.ts?